node/api/README.md

```diff
@@ -64,14 +67,171 @@ connections will be queued. The queue has a maximum size and when it is
+   'code': <ErrorCode>,
+   'message': <ErrorMessage>,
+   'data': [ ... ],
+   'request': { ... }
+  }
+ }
+ ```
+
+The codes and error messages include those specified in the [JSON-RPC 2.0
+ Specification](https://www.jsonrpc.org/specification#error_object) [1], some used (or proposed) [by Ethereum](https://github.com/ethereum/wiki/wiki/JSON-RPC-Error-Codes-Improvement-Proposal) [2], and some Aeternity-specific [3].
+
+ | Code   | Message            | Origin |
+ | ------ | ------------------ | ------ |
+ | -32700 | Parse error        | [1]    |
+ | -32000 | Invalid request    | [1]    |
+ | -32601 | Method not found   | [1]    |
+ | -32602 | Invalid params     | [1]    |
+ | -32603 | Internal error     | [1]    |
+ |      1 | Unauthorized       | [2]    |
+ |      2 | Action not allowed | [2]    |
+ |      3 | Rejected           | [2]    |
+
+The `'data'` element may contain more detailed information inside `'data'`:
+
+ | Code   | Message                   | Parent Code | Origin |
+ | ------ | ------------------------- | ----------- | ------ |
+ | 100    | X doesn't exist           |  3          |  [2]   |
+ | 101    | Requires coin             |  3          |  [2]   |
+ | 102    | Gas too low               |  3          |  [2]   |
+ | 103    | Gas limit exceeded        |  3          |  [2]   |
+ | 104    | Rejected                  |  3          |  [2]   |
+ | 105    | Value too low             |  3          |  [2]   |
+ | 106    | Timeout                   |  3          |  [2]   |
+ | 107    | Conflict                  |  3          |  [2]   |
+ | 1001   | Insufficient balance      |  3          |  [3]   |
+ | 1002   | Negative amount           |  3          |  [3]   |
+ | 1003   | Invalid pubkeys           |  3          |  [3]   |
+ | 1004   | Call not found            |  3          |  [3]   |
+ | 1005   | Broken encoding: accounts |  3          |  [3]   |
+ | 1006   | Broken encoding: contracts|  3          |  [3]   |
+ | 1007   | Contract init failed      |  3          |  [3]   |
+ | 1008   | Not a number              |  3          |  [3]   |
 
-| Name | Type | Description | Required |
-| ---- | ---- | ----------- | -------- |
-| action | string | what is the action | Yes |
-| tag | string | additional info for the action | No |
-| payload | object | data for action | Yes |
+#### Example, error object
+ ```javascript
+{
+  "error": {
+    "code": 3,
+    "data": [
+      {
+        "code": 1004,
+        "message": "Call not found"
+      }
+    ],
+    "message": "Rejected",
+    "request": {
+      "jsonrpc": "2.0",
+      "method": "channels.get.contract_call",
+      "params": {
+        "caller": "ak_KcR46Qk9aEmHnWwGZLAW7u5MT3FLsxszmkx9rphLfCZoBAzFY",
+        "contract": "ct_28pwhjFEVfuirjPzJDRTE4o4ybjKw7vjMymRdhhVrDEVTuXAtq",
+        "round": 8
+      }
+    }
+  },
+  "id": null,
+  "jsonrpc": "2.0"
+}
+```
+Detailed message transcripts from test suites can be found [here](./examples/aehttp_integration_SUITE/json-rpc/).
```

Comment:
The link seems not working for me, maybe you meant the following?
```suggestion
Detailed message transcripts from test suites can be found [here](./examples/channels/json-rpc/).
```

Reply:
Yes, sorry.